llsd/serde_binary.py

```diff
@@ -100,7 +100,7 @@ def _parse_map(self):
         count = 0
         cc = self._getc()
         key = b''
-        self._depth = self._depth + 1
+        self._depth += 1
```

Comment:
I don't know how much difference this makes, but I count 14 other instances of

    self._depth = self._depth [+-] 1